src/stactools/sentinel2/constants.py

```diff
@@ -31,79 +31,79 @@
     'B01':
     Band.create(name='B01',
                 common_name='coastal',
-                description='Band 1 - Coastal aerosol',
+                description='Coastal aerosol (band 1)',
                 center_wavelength=0.443,
                 full_width_half_max=0.027),
     'B02':
     Band.create(name='B02',
                 common_name='blue',
-                description='Band 2 - Blue',
+                description='Blue (band 2)',
                 center_wavelength=0.490,
                 full_width_half_max=0.098),
     'B03':
     Band.create(name='B03',
                 common_name='green',
-                description='Band 3 - Green',
+                description='Green (band 3)',
                 center_wavelength=0.560,
                 full_width_half_max=0.045),
     'B04':
     Band.create(name='B04',
                 common_name='red',
-                description='Band 4 - Red',
+                description='Red (band 3)',
                 center_wavelength=0.665,
                 full_width_half_max=0.038),
     'B05':
     Band.create(name='B05',
                 common_name='rededge',
-                description='Band 5 - Vegetation red edge 1',
```

Comment:
I took out vegetation as that speaks more to it's use than to what it actually is.